pkgs/development/libraries/ncurses/default.nix

```diff
@@ -14,17 +14,17 @@
 
 stdenv.mkDerivation rec {
   # Note the revision needs to be adjusted.
-  version = "6.1-20190112";
+  version = "6.2";
   name = "ncurses-${version}" + lib.optionalString (abiVersion == "5") "-abi5-compat";
 
   # We cannot use fetchFromGitHub (which calls fetchzip)
   # because we need to be able to use fetchurlBoot.
   src = let
     # Note the version needs to be adjusted.
-    rev = "acb4184f8f69fddd052a3daa8c8675f4bf8ce369";
+    rev = "47d2fb4537d9ad5bb14f4810561a327930ca4280";
```

Comment:
can this use the tag, at least for 6.2? :grin:

https://github.com/mirror/ncurses/releases/tag/v6.2

Reply:
Done.